package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha .\test\routs\testDutiesRouts.js && mocha .\test\routs\testJusticeBoard.js"
```

Comment:
You don't need to specify each file, mocha knows how to go over your files by itself